go.mod

```diff
@@ -30,10 +30,11 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tyler-smith/go-bip39 v1.1.0 // indirect
 	github.com/urfave/cli/v2 v2.3.0
+	github.com/yourbasic/graph v0.0.0-20210606180040-8ecfec1c2869
```

Comment:
this is an import used on `txspool.go`, but this PR is not affecting this specific file, so I'm not sure why this is being added at this moment, anyway, I just wanted to point this out because it is something weird to me.

Reply:
Yup it's wierd xD
I believe that at some point I didn't propagated something well, or made a branch with uncommited changes or something like that